pkgs/tools/security/vaultwarden/webvault.nix

```
@@ -23,10 +37,10 @@ in buildNpmPackage rec {
     owner = "bitwarden";
     repo = "clients";
     rev = "web-v${lib.removeSuffix "b" version}";
-    hash = "sha256-lDDy1b1yfw3nZrwEEkpvh6xYucgn20XHsGACc45eb2w=";
+    hash = "sha256-695iCkFhPEyyI4ekbjsdWpxgPy+bX392/X30HyL4F4Y=";
   };
 
-  npmDepsHash = "sha256-RR8Ua41D9SXymiPuabOnIab3byu8DR63rOfdeTaQpy4=";
+  npmDepsHash = "sha256-IJ5JVz9hHu3NOzFJAyzfhsMfPQgYQGntDEDuBMI/iZc=";
 
   postPatch = ''
     ln -s ${bw_web_builds}/{patches,resources} ..
```

We could also do 
```suggestion
    cp -r ${bw_web_builds}/{patches,resources} ..
```
to fix this.

---

Good idea. My PR got also accepted, so with the next update it doesn't matter anymore.